react/package.json

```diff
@@ -1,16 +1,17 @@
 {
   "name": "vtex.render-runtime",
-  "version": "6.0.0",
+  "version": null,
   "license": "UNLICENSED",
   "dependencies": {
+    "apollo-link-persisted-queries": "^0.1.0",
     "apollo-cache-inmemory": "^1.1.5",
     "apollo-client": "^2.2.0",
     "apollo-link-http": "^1.3.2",
     "apollo-utilities": "^1.0.4",
     "classnames": "^2.2.5",
     "eventemitter3": "^3.0.0",
     "exenv": "^1.2.2",
-    "graphql": "^0.12.3",
+    "graphql": "^0.11.0",
```

Comment:
Why use an older version of `graphql`? It's important for us to be on the most recent version as possible.

Reply:
Because graphql 0.12.3 conflicted with other packages. Apparently, version 0.13.1 fix this issue